sources/liquid_templates/_klass_table.liquid

```diff
@@ -11,6 +11,8 @@
   {%- assign stereotype = " " -%}
 {%- endif -%}
 
+{%- if klass.has_guidance? -%}
```

Comment:
@kwkwan can you please make this simpler with an `if` statement switch at the last column? We don't want a copy of the whole table.

Reply:
As the number of columns are needed to specified (`cols` and `n+` for column span) in the template such as:
```
[cols="1a,1a,2a,1a"]
....
| AGT 型の定義 3+| 
...
```
It is already in simplest form.

Reply:
Thanks @kwkwan , understand the complications. This is fine.